utils.c

```diff
@@ -305,3 +307,59 @@ char *janus_address_to_ip(struct sockaddr *address) {
 	}
 	return addr ? g_strdup(addr) : NULL;
 }
+
+/* PID file management */
+static char *pidfile = NULL;
+static int pidfd = -1;
+static FILE *pidf = NULL;
+int janus_pidfile_create(const char *file) {
+	if(file == NULL)
+		return 0;
+	pidfile = g_strdup(file);
```

Comment:
It's freed in another method, ```janus_pidfile_remove```, which is the one we call before leaving Janus. In fact we need to keep the path to the PID file to remove it (```unlink(pidfile)```) at the end.